### ISSUE TITLE
pdfiumdraw with windows XP (win32)

### ISSUE THREAD

**Issue by amastm** [abenori/TeX2img#2]
Thank you for this toolkit!
Everything seems to work in windows XP except:
pdfiumdraw.exe : "not a valid win32 application"

**Comment by aminophen** [comment]
I don't have windows XP, and I'm not a maintainer; but please let me check your environment:

[The website](http://island.geocities.jp/loveinequality/#requirement) (currently, written in Japanese only) says ".NET Framework 3.5 or later is required for TeX2img for Windows". You say, you have it on windows XP, and TeX2img.exe runs on it? And, only pdfiumdraw.exe doesn't work?

**Comment by amastm** [comment]
Correct: I have Windows XP inside a virtual machine (using vmware player).
.Net 3.5 and 4 is also installed. pdfiumdraw.exe is not running (not a valid x86 application it says).
If I can build pdfiumdraw with a C++ compiler inside XP easily, let me know.

**Comment by abenori** [comment]
I hope the folllowing pdfiumdraw works also with XP: https://1drv.ms/u/s!AnAe-sROy6tPhso-ifYqsl-Y_bN8VA
